aten/src/ATen/cpu/vec256/vec256_int.h

```diff
@@ -95,25 +95,25 @@ struct Vec256<int64_t> : public Vec256i {
     return _mm256_cmpeq_epi64(values, other.values);
   }
   Vec256<int64_t> operator!=(const Vec256<int64_t>& other) const {
-    auto zero = _mm256_set1_epi64x(0);
+    static const auto one = _mm256_set1_epi64x(-1);
```

Comment:
Don't mark these variables static. That will generate global loads and make this code much slower.